app/controllers/api/v1/migrations/external_controller.rb

```diff
@@ -93,7 +94,7 @@ def create_room
 
           user = User.find_by(email: room_hash[:owner_email], provider: 'greenlight')
 
-          return render_error status: :bad_request unless user
+          return render_error(status: :bad_request, errors: 'The room owner does not exist.') unless user
```

Comment:
We can shorten that to an identifier like `RecordNotFound` or even send a status 404 and handle that on v2.